### PR TITLE
adding file for Austin, TX

### PR DESCRIPTION
**Pull request by ireeve**
Pull request adds a file for things to do and places to eat in Austin Texas

#7

### CODE REVIEW

File: ireeve-austin.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Austin, TX 
+
+### Great Places to Eat
```
Comment (ireeve) [inline ireeve-austin.md]: Melissa likes Peanut butter ice cream

File: ireeve-austin.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Austin, TX
```
Comment (ireeve) [inline ireeve-austin.md]: re you sure you live here?

File: ireeve-austin.md (hunk 1 above)
Comment (ireeve) [inline ireeve-austin.md]: ok add peanut butter ice cream.

Comment (dunbam01) [inline ireeve-austin.md]: Pancakes with strawberries and whipped cream.

Comment (ireeve) [inline ireeve-austin.md]: Sounds good

**Comment by dunbam01** [comment]
French toast

**Comment by ireeve** [comment]
@dunbam01 - can you approve my pull request

**Pull request merged**